pkg/management/postgres/webserver/remote.go

```diff
@@ -170,8 +196,7 @@ func (ws *remoteWebserverEndpoints) pgStatus(w http.ResponseWriter, _ *http.Requ
 		return
 	}
 
-	w.Header().Set("Content-Type", "application/json")
-	_, _ = w.Write(js)
+	sendJSONResponseWithData(w, http.StatusOK, js)
```

Comment:
I'm hoping this will provide us some insight into getting no response, despite seeing the `"Instance status probe succeeding"` trace.